exercises/perfect-numbers/perfect_numbers_test.rb

```diff
@@ -0,0 +1,44 @@
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'example'
+
+class PerfectNumberTest < Minitest::Test
+  def test_aliquot_sum_zero
+    assert_equal 0, PerfectNumber.new(1).aliquot_sum
```

Comment:
I looked at the other implementations for reference and was a bit confused on this point. The lua variant has it similar to this where it tests both aliquot_sum and classify. Would you suggest just testing cases for classify and not aliquot_sum?